orchestra/project_api/views.py

```diff
@@ -124,3 +125,29 @@ def assign_worker_to_task(request):
         'success': success,
         'errors': errors,
     }
+
+
+@api_endpoint(methods=['POST'],
+              permissions=(IsSignedUser,),
+              logger=logger,
+              auths=(OrchestraProjectAPIAuthentication,))
+def message_project_team(request):
+    """
+    Endpoint for sending arbitrary message to a project team.
+    Payload example:
+    {'message': 'Chat message', 'project_id': 'some-id-123'}
+    """
+    data = load_encoded_json(request.body)
+    try:
+        message = data['message']
+        project_id = data['project_id']
+        project = Project.objects.get(id=project_id)
+    except KeyError:
+        text = ('An object with `message` and `project_id` attributes'
+                ' should be supplied')
+        raise BadRequest(text)
+    except Project.DoesNotExist:
+        raise BadRequest('No project for given id')
+    slack = OrchestraSlackService()
```

Comment:
Do we need to check that `slack_group_id` is not null?

Reply:
We should! I'll update the PR.